voronoi_game.py

```diff
@@ -271,9 +271,9 @@ def play_day(self, day):
                     map_states=self.map_states[day][0],
                     current_scores=self.player_score[day][0],
                     total_scores=self.player_total_score[day])
-                returned_action = [(float(dist), float(angle)) for dist, angle in returned_action]
```

Comment:
Moved this after check_action, as check_action verifies data format is correct first